test/orchard/inspect_test.clj

```diff
@@ -447,9 +447,9 @@
 
 (deftest path-test
   (testing "inspector tracks the path in the data structure"
```

Comment:
This test was frequently failing on a local wip branch after some changes to inspect/down fn. Changing it to a sorted-map to make debugging easier.